Beams/Testing/Cxx/vtkSlicerIECTransformLogicTest1.cxx

```diff
@@ -244,19 +337,381 @@ int vtkSlicerIECTransformLogicTest1(int vtkNotUsed(argc), char* vtkNotUsed(argv)
+    iecLogic->GetTransformNodeBetween(vtkSlicerIECTransformLogic::TableTop, vtkSlicerIECTransformLogic::TableTopEccentricRotation);
+  vtkTransform* tableTopEccentricRotationToPatientSupportTransform_Lateral_1 = vtkTransform::SafeDownCast(
+    tableTopToTableTopEccentricRotationTransformNode_Lateral_1->GetTransformToParent());
+
+
+  vtkNew<vtkMatrix4x4> tableTopEccentricRotationToPatientSupportMatrix_Lateral_1;
+  tableTopEccentricRotationToPatientSupportMatrix_Lateral_1->SetElement(0, 3, 1);
+  tableTopEccentricRotationToPatientSupportMatrix_Lateral_1->SetElement(1, 3, 0);
+  tableTopEccentricRotationToPatientSupportMatrix_Lateral_1->SetElement(2, 3, 0);
+  tableTopEccentricRotationToPatientSupportTransform_Lateral_1->SetMatrix(tableTopEccentricRotationToPatientSupportMatrix_Lateral_1);
+  tableTopEccentricRotationToPatientSupportTransform_Lateral_1->Modified();
+
+  double expectedTableTopToTableTop_Lateral_1_MatrixElements[16] =
+  { 1, 0, 0, 1,   0, 1, 0, 0,   0, 0, 1, 0,   0, 0, 0, 1 };
+  if (!IsTransformMatrixEqualTo(mrmlScene,
+    iecLogic->GetTransformNodeBetween(vtkSlicerIECTransformLogic::TableTop, vtkSlicerIECTransformLogic::TableTopEccentricRotation),
+    expectedTableTopToTableTop_Lateral_1_MatrixElements))
+  {
+    std::cerr << __LINE__ << ": PatientToTableTopTransform (lateral) does not match baseline" << std::endl;
+    return EXIT_FAILURE;
+  }
+
+  // Table Top (lateral), 230.0mm
+  vtkMRMLLinearTransformNode* tableTopToTableTopEccentricRotationTransformNode_Lateral_230 =
+    iecLogic->GetTransformNodeBetween(vtkSlicerIECTransformLogic::TableTop, vtkSlicerIECTransformLogic::TableTopEccentricRotation);
+  vtkTransform* tableTopEccentricRotationToPatientSupportTransform_Lateral_230 = vtkTransform::SafeDownCast(
+    tableTopToTableTopEccentricRotationTransformNode_Lateral_230->GetTransformToParent());
+
+
+  vtkNew<vtkMatrix4x4> tableTopEccentricRotationToPatientSupportMatrix_Lateral_230;
+  tableTopEccentricRotationToPatientSupportMatrix_Lateral_230->SetElement(0, 3, 230);
+  tableTopEccentricRotationToPatientSupportMatrix_Lateral_230->SetElement(1, 3, 0);
+  tableTopEccentricRotationToPatientSupportMatrix_Lateral_230->SetElement(2, 3, 0);
+  tableTopEccentricRotationToPatientSupportTransform_Lateral_230->SetMatrix(tableTopEccentricRotationToPatientSupportMatrix_Lateral_230);
+  tableTopEccentricRotationToPatientSupportTransform_Lateral_230->Modified();
+
+  double expectedTableTopToTableTop_Lateral_230_MatrixElements[16] =
+  { 1, 0, 0, 230,   0, 1, 0, 0,   0, 0, 1, 0,   0, 0, 0, 1 };
+  if (!IsTransformMatrixEqualTo(mrmlScene,
+    iecLogic->GetTransformNodeBetween(vtkSlicerIECTransformLogic::TableTop, vtkSlicerIECTransformLogic::TableTopEccentricRotation),
+    expectedTableTopToTableTop_Lateral_230_MatrixElements))
+  {
+    std::cerr << __LINE__ << ": PatientToTableTopTransform (lateral) does not match baseline" << std::endl;
+    return EXIT_FAILURE;
+  }
+
+  // Table Top (lateral), -230.0mm

```

Comment:
Extend comment explaining why we chose this number (that these are the min/max for the default Varian machine)